src/Microsoft.Azure.SignalR/HubHost/ServiceHubDispatcher.cs

```diff
@@ -17,17 +17,17 @@ internal class ServiceHubDispatcher<THub> where THub : Hub
     {
         private static readonly string Name = $"ServiceHubDispatcher<{typeof(THub).FullName}>";
 
-        private readonly ILoggerFactory _loggerFactory;
+        protected readonly ILoggerFactory _loggerFactory;
```

Comment:
no need to change it to `protected`? The mock dispatcher can assign it locally if need to